.github/workflows/test.yml

```diff
@@ -38,5 +38,3 @@ jobs:
 
   unit-tests:
     uses: vapor/ci/.github/workflows/run-unit-tests.yml@main
-    with:
-      with_coverage: true
```

Comment:
hmm why?

Reply:
Because this has been the default for some time now.